common/source/docs/common-escs-and-motors.rst

```diff
@@ -48,6 +48,7 @@ Protocols
     KDE CAN ESCs <common-kde-can-escs>
     DroneCAN ESCs <common-uavcan-escs>
     Currawong Velocity ESCs <common-velocity-can-escs>
+    Hobbywing DroneCAN ESCs <common-hobbywing-dronecan-esc>
```

Comment:
I prefer to  put this on the DroneCAN ESCs page...avoids flattening the hierarchy since its DroneCAN driver...you can link the new hobbywing-esc page from in a toctree...see below
also it should be titled Hobbywing DroneCAN Motor/ESC since its not just an ESC

Reply:
as discussed, we will leave the link on both pages for now.